Formula/nvm.rb

```diff
@@ -39,6 +39,6 @@ def caveats; <<-EOS.undent
     output = pipe_output("NODE_VERSION=homebrewtest #{prefix}/nvm-exec 2>&1")
     assert_no_match /No such file or directory/, output
     assert_no_match /nvm: command not found/, output
-    assert_match "N/A: version \"homebrewtest\" is not yet installed", output
+    assert_match /N\/A: version "N\/A" is not yet installed/, output
```

Comment:
There are additional statements after the "is not yet installed". The full output is:

```
N/A: version "N/A" is not yet installed.

You need to run "nvm install N/A" to install it before using it.
```